internal/cli/cmd/connectivity.go

```diff
@@ -162,15 +175,7 @@ func newCmdConnectivityTest(hooks Hooks) *cobra.Command {
 
 	cmd.Flags().StringSliceVar(&params.DeleteCiliumOnNodes, "delete-cilium-pod-on-nodes", []string{}, "List of node names from which Cilium pods will be delete before running tests")
 
-	cmd.Flags().BoolVar(&params.Perf, "perf", false, "Run network Performance tests")
```

Comment:
I think it would be a good idea to add a note on this field indicating that although it isn't user facing, it is still used to run the perf sub-command using items from the connectivity test suite.

Reply:
Makes sense, done!